tests/components/view_image_popover_bar.test.jsx

```diff
@@ -104,5 +104,14 @@ describe('components/view_image/popover_bar/PopoverBar', () => {
 
             expect(wrapper).toMatchSnapshot();
         });
+
+        test('should match CSS class on footer', () => {
```

Comment:
This test does not capture the issue where CSS class name is not present in the CSS file. Maybe it's time to use the styled-components so that we can verify not only the class name but also the style itself whether it's applied to HTML element or not.
I added this test just to make sure class name is applied to the main popover container.